templates/js-langgraph-agent/src/main.js

```diff
@@ -0,0 +1,97 @@
+import { HumanMessage } from '@langchain/core/messages';
+import { createReactAgent } from '@langchain/langgraph/prebuilt';
+import { ChatOpenAI } from '@langchain/openai';
+import { Actor, log } from 'apify';
+
+import { webSearchTool } from './tools.js';
+
+const Event = {
+    ACTOR_STARTED: 'actor-start-gb',
+    INPUT_TOKENS_COUNT_GPT4O: 'input-tokens-count-gpt-4o',
+    OUTPUT_TOKENS_COUNT_GPT4O: 'output-tokens-count-gpt-4o',
+    TASK_COMPLETED: 'task-completed',
+};
+
+await Actor.init();
+
+try {
+    const memoryMbytes = Actor.getEnv().memoryMbytes || 1024;
+    const memoryGB = Math.ceil(memoryMbytes / 1024);
+    log.info(`Required memory: ${memoryGB} GB. Charging Actor start event.`);
+    await Actor.charge({ eventName: Event.ACTOR_STARTED, count: memoryGB });
+} catch (error) {
+    log.error('Failed to charge for actor start event', { error });
+    await Actor.exit(1);
+}
+
+// Follow these steps to run this template:
+// 1. If running locally, authenticate to the Apify platform by executing `apify login` in your terminal.
+//    This is necessary to run the Website Content Crawler Actor for data gathering.
+// 2. Set the `OPENAI_API_KEY` environment variable with your OpenAI API key, which can be obtained from
+//    https://platform.openai.com/account/api-keys. Refer to
+//    https://docs.apify.com/cli/docs/vars#set-up-environment-variables-in-apify-console for guidance
+//    on setting environment variables.
+const { OPENAI_API_KEY, APIFY_TOKEN } = process.env;
+
+// You can configure the input for the Actor in the Apify UI when running on the Apify platform or editing
+// storage/key_value_stores/default/INPUT.json when running locally.
+const {
+    // query = 'How to build LangGraph agent at Apify platform?',
+    query = 'This is fallback test query, do nothing and ignore it.',
```

Comment:
Please clean up the comment.
If there's no query provided, wouldn't it be better to crash? It doesn't make sense to run the code without a valid query.